[RN] Add kommander 1.4.3 release notes (#3929)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.4/release-notes/1.4.3/index.md

[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]

To get started with Kommander, [download](/dkp/konvoy/1.8/download/) and [install](/dkp/konvoy/1.8/install/) the latest version of Konvoy.

<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>

# Release Summary
Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.

## Features/Improvements

- Updated Kubecost enterprise key

### Bug fixes

> - Fixed an issue where the Kommander UI could run out of ports. Kommander UI now monitors port usage and can recover automatically if it runs out. 

Reviewer: What's the cops that goes with this?

- Fixed an issue with attaching clusters with networking restrictions that also use an HTTP Proxy (COPS-7078).

- Resolved UI issue where socks requests were timing out.

### Component Versions

> - Addon: 1.4.3-2

Reviewer 2: This was bumped to `1.4.3-3` -[ in rc 2](https://github.com/mesosphere/kubeaddons-kommander/blob/v1.4.3-rc.2/addons/kommander/1.4/kommander.yaml#L12) 
from 
`1.4.3-2` - [in rc 1](https://github.com/mesosphere/kubeaddons-kommander/blob/v1.4.3-rc.1/addons/kommander/1.4/kommander.yaml#L12)

Reviewer 2: ```suggestion
- Addon: 1.4.3-3
```

> - Chart: 0.38.4

Author: ```suggestion
- Chart: 0.38.5
```

---
layout: layout.pug
beta: false
navigationTitle: Release Notes Kommander 1.4.3
title: Release Notes Kommander 1.4.3
menuWeight: 40
excerpt: View release-specific information for Kommander 1.4.3
enterprise: false
---

<!-- markdownlint-disable MD034 -->

**D2iQ&reg; Kommander&reg; version 1.4.3 was released on December 9, 2021.**

> [button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]

Reviewer 3: That's the wrong link, it needs to be:

```suggestion
[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
```

To get started with Kommander, [download](/dkp/konvoy/1.8/download/) and [install](/dkp/konvoy/1.8/install/) the latest version of Konvoy.

<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>

# Release Summary
Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.

## Features/Improvements

> - Updated Kubecost enterprise key

Reviewer 3: Does this have any user visibility or impact?   If not, I'd argue this is not really a feature or improvement.  If it does, we should focus on what 'feature/improvement' this adds to the user experience. 

@gracedo do you know?

Reviewer: agree there.

Reviewer 4: @cneth @cmastr I found the related ticket: https://jira.d2iq.com/browse/D2IQ-81160?jql=text%20~%20%22Kubecost%20enterprise%20key%22

Is the legacy userbase wide enough to include as @gracedo mentioned: "For legacy users using enterprise features of kubecost, we need to come up with a plan. Perhaps we can first get the key to unblock the 2.1 release, then invalidate the old key once we have a plan. It likely involves asking customers to upload the new key - or upgrading to a patch release."

Author: No cops here. Old key is still valid, I don't think we need to include any messaging for this now.

Author: ```suggestion
## Features/Improvements

- Updated Kubecost enterprise key.
```

### Bug fixes

> - Fixed an issue where Kommander UI runs out of ports. This monitors ports and forces a restart if it runs out. 

Reviewer 3: ```suggestion
- Fixed an issue where the Kommander UI could run out of ports.  Kommander UI now monitors port usage and can recover automatically if it runs out. 
```

> - Fixed http proxing support in unidirectional environments (COPS-7078).

Reviewer 3: 'unidirectional environments' is not a term we use in the documentation.   We call this "networking restrictions" (https://docs.d2iq.com/dkp/kommander/1.4/clusters/attach-cluster/cluster-with-network-restrictions/)

```suggestion
- Fixed an issue with attaching clusters with networking restrictions that also use an HTTP Proxy (COPS-7078).
```

Reviewer 4: This is interesting, so for my own education: the clusters contain networking restrictions or they're being attached to networking restrictions?

Reviewer 4: Nevermind, read accompanying docs. Committing and resolving.

> - Updated Traefik Forward Auth to 3.0.3, pulling in a fix for URL pattern matching.

Reviewer 3: Traefik Forward Auth is installed by Konvoy 1.x as part of the Kubeaddons, it's not part of Kommander 1.4.   This issue is already documented there:

https://github.com/mesosphere/konvoy/blob/release-1.8/docs/site/release-notes/kubernetes-base-addon/index.md?plain=1#L22

```suggestion
```

> - Resolved UI issue where socks requests were timing out.

Reviewer 3: Do we have any more details on what this means or what environments this might be experienced in? It seems like it might be related to proxies again, maybe it is part of the COPS-7078 stuff?

```suggestion
- Resolved a UI issue where socks requests were timing out.
```

Reviewer: Agree, I think this needs a COPS number. @mwrightd2 let's work together to track this down.
